code/modules/clothing/spacesuits/void/void.dm

```diff
@@ -191,23 +191,26 @@
 
 	set name = "Eject Voidsuit Tank"
 	set category = "Object"
-	set src in usr
+	set src in view(1)
 
-	if(!istype(src.loc,/mob/living)) return
+	if(!src.Adjacent(usr))	return
+
+	var/mob/living/carbon/human/H = usr
+
+	if(!istype(H) || H.stat) return
```

Comment:
You wanna use `use_check_and_message` here.

Reply:
Thank you and Dunzo